tests/e2e/api-core-tests/data/shared/batch-update.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Shared model for batch updates for a resource.
+ *
+ * Note that by default the update is limited to 100 objects to be created, updated, or deleted.
+ */
+const batchUpdatePayload = {
+	create: [],
+	update: [],
+	delete: [],
+}
+
+/**
+ * Batch create a resource.
+ *
+ * @param {Array} resourcesToCreate A list of resource objects to create.
+ * @returns
```

Comment:
`@returns` missing return type.

`batchCreate`, `batchUpdate`, and `batchDelete` are identical functions except the internal variable `{prefix}Array`. It looks like they could be replaced with a single function with validation logic and a spread operator 

```
// after validation
const xArray = [ ...resourcesToX ];
```

Reply:
That's a good idea. I played with this a bit and ended up deciding to simplify these down into a single `batch( action, resources = [], payload = {} )` method. Let me know what you think of this solution.